src/NUnitEngine/nunit.engine.core/Runners/AggregatingTestRunner.cs

```diff
@@ -55,7 +55,8 @@ public IList<ITestEngineRunner> Runners
                 if (_runners == null)
                 {
                     _runners = new List<ITestEngineRunner>();
-                    foreach (var subPackage in TestPackage.SubPackages)
+                    foreach (var subPackage in TestPackage.Select(p => !p.HasSubPackages()))
+                    //foreach (var subPackage in TestPackage.SubPackages)
```

Comment:
Was the comment meant to remain?

Reply:
Nope. I tested by alternating between the two expressions in several places. Forgot this one. I'll delete it.